test/integration/pkg_install_test.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+var distros = []string{
+	"debian:sid",
+	"debian:latest",
+	"debian:10",
+	"debian:9",
+	//
+	"ubuntu:latest",
+	"ubuntu:20.10",
+	"ubuntu:20.04",
+	"ubuntu:19.10",
+	"ubuntu:19.04",
+}
+
+var timeout = Minutes(10)
+
+
+// TestPackageInstall tests installation of .deb packages with minikube itself and with kvm2 driver
+//   on debian/ubuntu docker images enumerated in "distros"
+func TestDebPackageInstall(t *testing.T) {
+
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
```

Comment:
maybe limit this test on a machine that has 'docker installed" 
if docker not installed skipped

Reply:
Yep, good point. done.